### ISSUE TITLE
Custom History using too many resources

### ISSUE THREAD

**Issue by kilbot** [woothemes/woocommerce#43804]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

If a customer has a lot of orders, the Customer History metabox on the Order Edit page can cause the page to crash.

### Expected behavior

Expect Customer History to be cached and optimised for large number or orders.

### Actual behavior

Encountered an issue on a client's website where `WooCommerce > Orders > Edit` wouldn't load for one particular customer.

### Steps to reproduce

1. Create many orders for a customer (thousands?)
2. Try to open any order associated with that user
3. Page will eventually crash

<details><summary>WordPress Environment</summary>
<p>

```
### WordPress Environment ###

WordPress address (URL): https://www.wildandwoollyshop.co.uk
Site address (URL): https://www.wildandwoollyshop.co.uk
WC Version: 8.5.1
REST API Version: ✔ 8.5.1
WC Blocks Version: ✔ 11.8.0-dev
Action Scheduler Version: ✔ 3.7.1
Log Directory Writable: ✔
WP Version: 6.4.2
WP Multisite: –
WP Memory Limit: 512 MB
WP Debug Mode: –
WP Cron: –
Language: en_US
External object cache: –

### Server Environment ###

Server Info: Apache/2.4.38 (Debian)
PHP Version: 7.4.33
PHP Post Max Size: 65 MB
PHP Time Limit: 120
PHP Max Input Vars: 1000
cURL Version: 7.64.0
OpenSSL/1.1.1d

SUHOSIN Installed: –
MySQL Version: 5.5.5-10.5.15-MariaDB-0+deb11u1-log
Max Upload Size: 64 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 8.5.1
WC Database Prefix: wp_
Total Database Size: 918.47MB
Database Data Size: 562.64MB
Database Index Size: 355.83MB
wp_woocommerce_sessions: Data: 9.02MB + Index: 1.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wp_woocommerce_downloadable_product_permissions: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_woocommerce_order_items: Data: 7.66MB + Index: 4.81MB + Engine MyISAM
wp_woocommerce_order_itemmeta: Data: 84.14MB + Index: 57.79MB + Engine MyISAM
wp_woocommerce_tax_rates: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
wp_woocommerce_tax_rate_locations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
woof_query_cache: Data: 0.31MB + Index: 0.28MB + Engine InnoDB
wp_actionscheduler_actions: Data: 8.06MB + Index: 4.70MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 20.09MB + Index: 12.53MB + Engine InnoDB
wp_atum_order_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_atum_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_atum_product_data: Data: 0.30MB + Index: 0.31MB + Engine InnoDB
wp_commentmeta: Data: 0.04MB + Index: 0.03MB + Engine MyISAM
wp_comments: Data: 31.28MB + Index: 11.68MB + Engine MyISAM
wp_create_map: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_group_map: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_jetpack_sync_queue: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_links: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_map_locations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_mw_wc_qbo_sync_customer_pairs: Data: 0.06MB + Index: 0.00MB + Engine InnoDB
wp_mw_wc_qbo_sync_log: Data: 1.02MB + Index: 0.00MB + Engine InnoDB
wp_mw_wc_qbo_sync_paymentmethod_map: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mw_wc_qbo_sync_payment_id_map: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mw_wc_qbo_sync_product_pairs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mw_wc_qbo_sync_qbo_customers: Data: 0.30MB + Index: 0.00MB + Engine InnoDB
wp_mw_wc_qbo_sync_qbo_items: Data: 0.19MB + Index: 0.00MB + Engine InnoDB
wp_mw_wc_qbo_sync_real_time_sync_history: Data: 2.52MB + Index: 0.00MB + Engine InnoDB
wp_mw_wc_qbo_sync_real_time_sync_queue: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mw_wc_qbo_sync_sessions: Data: 0.14MB + Index: 0.05MB + Engine InnoDB
wp_mw_wc_qbo_sync_shipping_product_map: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mw_wc_qbo_sync_tax_map: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_mw_wc_qbo_sync_variation_pairs: Data: 0.04MB + Index: 0.02MB + Engine MyISAM
wp_mw_wc_qbo_sync_wq_cf_map: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_my_calendar: Data: 0.01MB + Index: 0.00MB + Engine MyISAM
wp_my_calendar_categories: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_my_calendar_events: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_my_calendar_locations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
wp_newsletter: Data: 2.52MB + Index: 0.41MB + Engine InnoDB
wp_newsletter_emails: Data: 6.25MB + Index: 0.00MB + Engine InnoDB
wp_newsletter_sent: Data: 7.52MB + Index: 4.03MB + Engine InnoDB
wp_newsletter_stats: Data: 12.52MB + Index: 8.03MB + Engine InnoDB
wp_newsletter_user_logs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_newsletter_user_meta: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_options: Data: 20.71MB + Index: 1.38MB + Engine MyISAM
wp_pmxi_files: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_history: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_images: Data: 0.19MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_imports: Data: 0.09MB + Index: 0.00MB + Engine InnoDB
wp_pmxi_posts: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_pmxi_templates: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_popup_banners: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_postmeta: Data: 129.85MB + Index: 81.01MB + Engine MyISAM
wp_posts: Data: 19.94MB + Index: 5.09MB + Engine MyISAM
wp_pos_warehouse: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_stock_log: Data: 2.52MB + Index: 0.00MB + Engine InnoDB
wp_tec_events: Data: 0.06MB + Index: 0.02MB + Engine InnoDB
wp_tec_occurrences: Data: 0.08MB + Index: 0.03MB + Engine InnoDB
wp_tec_series_relationships: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_termmeta: Data: 0.14MB + Index: 0.20MB + Engine InnoDB
wp_terms: Data: 0.02MB + Index: 0.06MB + Engine MyISAM
wp_term_relationships: Data: 0.12MB + Index: 0.28MB + Engine MyISAM
wp_term_taxonomy: Data: 0.03MB + Index: 0.04MB + Engine MyISAM
wp_usermeta: Data: 7.37MB + Index: 4.81MB + Engine MyISAM
wp_users: Data: 0.42MB + Index: 0.47MB + Engine MyISAM
wp_vtprd_purchase_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_vtprd_purchase_log_product: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_vtprd_purchase_log_product_rule: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_vxc_qbooks_accounts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_vxc_qbooks_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.09MB + Index: 0.00MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.11MB + Index: 0.02MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 1.52MB + Index: 1.95MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_orders: Data: 9.52MB + Index: 14.59MB + Engine InnoDB
wp_wc_orders_meta: Data: 39.58MB + Index: 91.42MB + Engine InnoDB
wp_wc_order_addresses: Data: 6.52MB + Index: 11.06MB + Engine InnoDB
wp_wc_order_bundle_lookup: Data: 0.09MB + Index: 0.09MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.06MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_operational_data: Data: 5.52MB + Index: 4.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 10.52MB + Index: 12.05MB + Engine InnoDB
wp_wc_order_stats: Data: 8.52MB + Index: 4.50MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 2.52MB + Index: 3.03MB + Engine InnoDB
wp_wc_product_attributes_lookup: Data: 0.09MB + Index: 0.09MB + Engine InnoDB
wp_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 1.52MB + Index: 0.92MB + Engine InnoDB
wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.11MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wdr_order_discounts: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wdr_order_item_discounts: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wdr_rules: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfblockediplog: Data: 0.28MB + Index: 0.00MB + Engine InnoDB
wp_wfblocks7: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wfconfig: Data: 2.25MB + Index: 0.00MB + Engine InnoDB
wp_wfcrawlers: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wffilechanges: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wffilemods: Data: 54.64MB + Index: 0.00MB + Engine InnoDB
wp_wfhits: Data: 2.02MB + Index: 0.22MB + Engine InnoDB
wp_wfhoover: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wfissues: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wfknownfilelist: Data: 22.55MB + Index: 0.00MB + Engine InnoDB
wp_wflivetraffichuman: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wflocs: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wflogins: Data: 1.02MB + Index: 0.14MB + Engine InnoDB
wp_wfls_2fa_secrets: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wfls_role_counts: Data: 0.00MB + Index: 0.00MB + Engine MEMORY
wp_wfls_settings: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfnotifications: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfpendingissues: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wfreversecache: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfsecurityevents: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfsnipcache: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wfstatus: Data: 2.02MB + Index: 2.03MB + Engine InnoDB
wp_wftrafficrates: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wfwaffailures: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_bundled_itemmeta: Data: 0.09MB + Index: 0.09MB + Engine InnoDB
wp_woocommerce_bundled_items: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_extra_flat_rates: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpfm_backup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpgmza: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpgmza_categories: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpgmza_category_maps: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpgmza_maps: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpgmza_polygon: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wpgmza_polylines: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_yoast_indexable: Data: 12.36MB + Index: 8.33MB + Engine InnoDB
wp_yoast_indexable_hierarchy: Data: 1.22MB + Index: 0.89MB + Engine InnoDB
wp_yoast_migrations: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_yoast_primary_term: Data: 0.06MB + Index: 0.03MB + Engine InnoDB
wp_yoast_seo_links: Data: 0.52MB + Index: 0.23MB + Engine InnoDB
wp_yoast_seo_meta: Data: 0.45MB + Index: 0.00MB + Engine InnoDB

### Post Type Counts ###

acf-field: 18
acf-field-group: 3
attachment: 10500
atum_supplier: 37
customize_changeset: 1
custom_css: 1
cwginstocknotifier: 245
download: 11
imagemap: 2
imagemap_area: 16
mc-events: 23
mc4wp-form: 1
nav_menu_item: 67
oembed_cache: 2
page: 35
popup: 4
popup_theme: 8
post: 166
pos_custom_product: 2
pos_temp_register_or: 5
product: 607
product_variation: 3760
revision: 1915
rp_shop_giftcard: 269
shipping_option: 1
shop_coupon: 295
shop_order: 38120
shop_order_refund: 776
stores: 1
tribe_events: 185
tribe_event_series: 3
tribe_organizer: 5
tribe_rsvp_attendees: 22
tribe_rsvp_tickets: 5
tribe_tpp_orders: 4
tribe_venue: 9
tribe_wooticket: 1081
vxc_qbooks: 1
wcps: 2
wcps_layout: 3
wc_afrsm: 2
wc_order_status: 10
woocustomemails: 7
wpsc-product: 69
wp_global_styles: 1
wp_navigation: 1
wp_template: 4

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (45) ###

Advanced Custom Fields PRO: by WP Engine – 6.2.5
Back In Stock Notifier for WooCommerce | WooCommerce Waitlist Pro: by codewoogeek – 5.1.0
Custom Related Products for WooCommerce: by Scott Nelle – 1.3
Event Tickets Plus: by The Events Calendar – 5.8.1
Event Tickets: by The Events Calendar – 5.7.1
The Events Calendar Pro: by The Events Calendar – 6.2.4
Facebook for WooCommerce: by Facebook – 3.1.9
Insert Pages: by Paul Ryan – 3.7.7
Jetpack: by Automattic – 13.0
WooCommerce Sync for QuickBooks Online - by MyWorks Software: by MyWorks Software – 2.8.1
Newsletter - Extended Composer Blocks: by The Newsletter Team – 1.5.1
Newsletter - Addons Manager: by The Newsletter Team – 1.2.0
Newsletter - Import: by The Newsletter Team – 1.4.4
Newsletter - SMTP Addon: by The Newsletter Team – 1.0.9
Newsletter - The Events Calendar Addon: by The Newsletter Team – 1.2.9
Newsletter - WooCommerce: by The Newsletter Team – 1.8.6
Newsletter: by Stefano Lissa & The Newsletter Team – 8.0.7
Post Thumbnail Editor: by sewpafly – 2.4.8
Product/Review CSV Import Export: by WebToffee – 3.8.3
Product Open Pricing (Name Your Price) for WooCommerce: by WP Wham – 1.7.1
SSH SFTP Updater Support: by TerraFrost
David Anderson + Team Updraft – 0.8.5

The Events Calendar: by The Events Calendar – 6.2.9
Theme Switcha: by Jeff Starr – 3.3.2
Woo Discount Rules: by Flycart – 2.6.2
WooCommerce Cost of Goods: by SkyVerge – 2.13.0
Print Invoice & Delivery Notes for WooCommerce: by Tyche Softwares – 4.8.1
WooCommerce Direct Variation Link: by WP BackOffice – 1.0.3
WooCommerce Stripe Gateway: by WooCommerce – 7.9.1
WooCommerce PayPal Payments: by WooCommerce – 2.4.3
WooCommerce POS Pro: by kilbot – 1.4.2
WooCommerce POS: by kilbot – 1.4.7
WooCommerce Prices: by sydcode – 1.0.1
WooCommerce Product Bundles: by WooCommerce – 6.22.5 (update to version 6.22.6 is available)
Product Slider for WooCommerce by PickPlugins: by PickPlugins – 1.13.49
WooCommerce Smart Coupons: by StoreApps – 8.13.0
Stock Manager for WooCommerce: by StoreApps – 3.0.0
WooCommerce: by Automattic – 8.5.1
Wordfence Assistant: by Wordfence – 1.0.9
Wordfence Login Security: by Wordfence – 1.1.8
Wordfence Security: by Wordfence – 7.11.1
Yoast SEO: by Team Yoast – 21.8
WP Crontrol: by John Blackbourn & crontributors – 1.16.1
WP Super Cache: by Automattic – 1.11.0
WP Fortune: by WP Fortune – 1.0.8
XT Variation Swatches for WooCommerce Pro: by XplodedThemes – 1.9.0

### Inactive Plugins (8) ###

Advanced Custom Fields: by WP Engine – 6.2.5
Media Sync: by Media Sync Team – 1.4.0
Nelio A/B Testing: by Nelio Software – 6.3.3
Newsletter - Archive: by The Newsletter Team – 4.1.1
Newsletter - Instasend (BETA): by The Newsletter Team – 1.0.3
Newsletter - Reports and Retargeting: by The Newsletter Team – 4.5.4
Popup Maker: by Popup Maker – 1.18.2
WAW custom: by  –

### Dropin Plugins (1) ###

advanced-cache.php: advanced-cache.php

### Must Use Plugins (1) ###

Health Check Troubleshooting Mode: by  – 1.9.1

### Settings ###

API Enabled: –
Force SSL: –
Currency: GBP (£)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: bundle (bundle)
external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to Woo.com: ✔
Enforce Approved Product Download Directories: –
HPOS feature screen enabled: –
HPOS feature enabled: –
Order datastore: WC_Order_Data_Store_CPT
HPOS data sync enabled: ✔

### WC Pages ###

Shop base: #729 - /shop/
Cart: #731 - /cart/
Checkout: #733 - /checkout/
My account: #735 - /my-account/
Terms and conditions: #3698 - /returns/

### Theme ###

Name: Wild and Woolly 2016
Version: 1.0.0
Author URL: http://sarahmacbeth.com
Child Theme: ✔
Parent Theme Name: Arcade Basic
Parent Theme Version: 1.1.1
Parent Theme Author URL: http://bavotasan.com/
WooCommerce Support: ✔

### Templates ###

Archive Template: Your theme has a woocommerce.php file
you will not be able to override the woocommerce/archive-product.php custom template since woocommerce.php has priority over archive-product.php. This is intended to prevent display issues.

Overrides: wild-and-woolly/content-product.php
wild-and-woolly/woocommerce/emails/customer-completed-order.php version 3.5.0 is out of date. The core version is 3.7.0
wild-and-woolly/woocommerce/single-product/add-to-cart/external.php
wild-and-woolly/woocommerce/single-product/add-to-cart/grouped.php
wild-and-woolly/woocommerce/single-product/add-to-cart/simple.php version 3.4.0 is out of date. The core version is 7.0.1
wild-and-woolly/woocommerce/single-product/add-to-cart/variable.php
wild-and-woolly/woocommerce/single-product/add-to-cart/variation-add-to-cart-button.php version 3.4.0 is out of date. The core version is 7.0.1
wild-and-woolly/woocommerce/single-product/add-to-cart/variation.php
wild-and-woolly/woocommerce/single-product/product-attributes.php version 3.4.0 is out of date. The core version is 3.6.0

Outdated Templates: ❌
					
					
						Learn how to update

### Product Open Pricing Settings ###

Product Open Pricing (Name Your Price): yes
Frontend template: Choose Amount %open_price_input% %currency_symbol%
Input field position: woocommerce_before_add_to_cart_quantity
Position priority.: 9999
Hide price: yes
Disable quantity selector: yes
Display on loop: no
Frontend loop template: Choose your amount %open_price_input% %currency_symbol%
Input field loop position: woocommerce_after_shop_loop_item
Position priority.: 9
Hide price: yes
Input style: width:75px;text-align:center;
Input pattern: 
Enable step ticker: yes
Price step: 0.01
Force number of decimals: no
Number of decimals.: 2
Add To Cart button text: 
Message on empty price: Price is required!
Message on price too low: Price is too low!
Message on price too high: Price is too high!
Add "Open Pricing" column: no
Fix mini cart: no

### Product Bundles ###

Database Version: 6.22.5
Loopback Test: ✔
Template Overrides: –

### WooCommerce PayPal Payments ###

Onboarded: ✔
Shop country code: GB
WooCommerce currency supported: ✔
Advanced Card Processing available in country: ✔
Pay Later messaging available in country: ✔
Webhook status: ✔
PayPal Vault enabled: –
ACDC Vault enabled: –
Logging enabled: –
Reference Transactions: –
Used PayPal Checkout plugin: ✔
Subscriptions Mode: Disabled

### Admin ###

Enabled Features: activity-panels
analytics
product-block-editor
coupons
core-profiler
customer-effort-score-tracks
import-products-task
experimental-fashion-sample-products
shipping-smart-defaults
shipping-setting-tour
homescreen
marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
product-variation-management
product-virtual-downloadable
product-external-affiliate
product-grouped
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
woo-mobile-welcome
wc-pay-promotion
wc-pay-welcome-page

Disabled Features: customize-store
minified-js
new-product-management-experience
product-linked
settings
async-product-editor-category-field

Daily Cron: ✔ Next scheduled: 2024-01-19 12:12:52 +00:00
Options: ✔
Notes: 197
Onboarding: completed

### Action Scheduler ###

Canceled: 1
Oldest: 2024-01-13 23:29:31 +0000
Newest: 2024-01-13 23:29:31 +0000

Complete: 11,641
Oldest: 2023-12-19 00:33:13 +0000
Newest: 2024-01-19 00:17:41 +0000

Failed: 123
Oldest: 2019-09-17 12:45:00 +0100
Newest: 2024-01-19 00:19:04 +0000

Pending: 6
Oldest: 2024-01-19 00:22:41 +0000
Newest: 2024-01-25 09:04:10 +0000


### Smart Coupons related settings ###

Number of coupons to show: 5
Number of characters in auto-generated coupon code: 
Valid order status for auto-generating coupon: processing
completed

Include tax in the amount of the generated gift card: no
Deduct credit/gift before doing tax calculations: no
Gift Card discount is inclusive of tax: no
Automatic deletion: no
Coupon emails: yes
Printing coupons: yes
Sell gift cards at less price?: no
Use gift card applied in first subscription order for subsequent renewals until credit reaches zero: yes
Renewal orders should not generate coupons even when they include a product that issues coupons: no
Allow sending of coupons to others: yes
Allow schedule sending of coupons?: yes
Combine emails: yes
Auto generated coupon email: yes
Combined auto generated coupons email: yes
Acknowledgement email: yes
Enable taxes: yes
Prices entered with tax: yes
Rounding: no
Display prices in the shop: incl
Display prices during cart and checkout: incl
Display tax totals: single
Enable the use of coupon codes: yes
Calculate coupon discounts sequentially: yes
Account endpoints > Coupons: wc-smart-coupons

### Status report information ###

Generated at: 2024-01-19 00:20:44 +00:00
```

</p>
</details> 

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woo.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by E1NSER** [comment]
Same issue here

**Comment by layoutd** [comment]
Hi @kilbot and @E1NSER, this should have been resolved with #44318 and shipped in WooCommerce 8.6. 

Have you updated and, if so, can you confirm the issue no longer occurs?

**Comment by layoutd** [comment]
Closing this issue, we can reopen if it needs to be addressed further. Thanks again for reporting and commenting!
